services/crypto_update.proto

```diff
@@ -155,16 +155,4 @@ message CryptoUpdateTransactionBody {
      * If true, the account declines receiving a staking reward. The default value is false.
      */
     google.protobuf.BoolValue decline_reward = 18;
-
-    oneof virtual_address_update {
-        /**
-         * The virtual address to be added.
-         */
-        VirtualAddress add = 20;
```

Comment:
Can you also remove `VirtualAddress` from `basic_types.proto`

Reply:
`VirtualAddress` was removed `basic_types.proto` from in a previous commit

Reply:
I missed that, thanks